crates/rolldown_common/src/types/package_json.rs

```diff
@@ -0,0 +1,17 @@
+use std::sync::Arc;
+
+#[derive(Debug)]
+pub struct PackageJson {
```

Comment:
I use a custom `PackageJson` instead of oxc's. Becuase the fields we care about is different from oxc.

Reply:
We need realpath from `oxc` real path, I am gonna revert this PackageJson, 
https://github.com/rolldown/rolldown/pull/1103/files#diff-c9c7714142367c1f6a60995eb1e3ec2255fcab1aa62141a33de5268a814bdf57R317-R321

Reply:
Could you just assign the realpath to our `PackageJson`?

Reply:
Using our own `PackageJson` is friendly for adding extra fields in the future.

Reply:
sure, that's what I actually did.